From 7b4dea0cd7def5f994c924f353f1f536a4c547c1 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Cl=C3=A9ment=20Le=20Goffic?= <clement.legoffic@foss.st.com>
Date: Tue, 29 Apr 2025 15:25:56 +0200
Subject: [PATCH] dt-bindings: remoteproc: fix stm32-m0 example
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Add missing include in the example.

Change-Id: Ie5486b4036e1345482af2cd1691d60705e367f26
Signed-off-by: Clément Le Goffic <clement.legoffic@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/455193
Reviewed-by: Arnaud POULIQUEN <arnaud.pouliquen@st.com>
Reviewed-by: Amelie DELAUNAY <amelie.delaunay@foss.st.com>
Tested-by: Amelie DELAUNAY <amelie.delaunay@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Domain-Review: Arnaud POULIQUEN <arnaud.pouliquen@st.com>
---
 Documentation/devicetree/bindings/remoteproc/st,stm32-m0.yaml | 2 ++
 1 file changed, 2 insertions(+)

--- a/Documentation/devicetree/bindings/remoteproc/st,stm32-m0.yaml
+++ b/Documentation/devicetree/bindings/remoteproc/st,stm32-m0.yaml
@@ -72,7 +72,9 @@ additionalProperties: false
 
 examples:
   - |
+    #include <dt-bindings/clock/st,stm32mp25-rcc.h>
     #include <dt-bindings/reset/st,stm32mp25-rcc.h>
+
     m0_rproc: m0@0 {
       compatible = "st,stm32mp2-m0";
       reg = <0 0>;
